programmers/크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,24 @@
+function solution(board, moves) {
+    
+    // const newMoves = moves.map((x) =>  x - 1);
+    let count = 0;
+    let stack = [];
```

Comment:
```suggestion
    const stack = [];
```
배열 전체를 바꿀께 아니라면 const로 지정해주시는게 좋아요